### PR TITLE
fix some typos, link layout, and translations in Japanese `Install` page.

### LINKED ISSUE #423: Japanese Install page has some link issues
I realized the Japanese `Install` page of current master has some link issues:
SS 2021-04-24 at 20 59 55
I think it should be fixed.

### PR DESCRIPTION
**Pull request by AtsushiSakai**
#### Brief description of what is fixed or changed

I fixed some typos, link layout, and translations in Japanese `Install` page to fix #423

### CODE REVIEW

**Comment by AtsushiSakai** [comment]
@kngwyu I'm happy for you to review this PR and whole `Install` page if you have time.

**Comment by rgommers** [comment]
I've pushed some changes to bring this branch in sync with master. It is no longer necessary to touch `config.yaml` (was a generated file, deleted now) or to comment out `disabledLanguages`. The preview builds now always show translations; the deploy builds not yet.

Locally you can set an environment variable once: `export NUMPYORG_WITH_TRANSLATIONS = "1"`, and then building the site with:
```
python gen_config.py && hugo
```
will show the translations.

File: content/ja/install.md [hunk 1]
```diff
@@ -109,55 +109,29 @@ Python パッケージをインストールするための2 つの主要なツ
 
 
 
-## Numpyパッケージ & 高速線形代数ライブラリ
+## Numpyパッケージと高速線形代数ライブラリ
```
Comment (kngwyu) [inline content/ja/install.md:112]:
```suggestion
## NumPyパッケージと高速線形代数ライブラリ
```

File: content/ja/install.md [hunk 2]
```diff
@@ -109,55 +109,29 @@ Python パッケージをインストールするための2 つの主要なツ
...
 
-Numpy は他の Python パッケージに依存していませんが、高速な線形代数ライブラリ - 一般的には、 [インテル® MKL](https://software.intel.com/en-us/mkl) または [OpenBLAS](https://www.openblas.net/) に依存しています。 ユーザーはこれらの線形代数ライブラリのインストールを心配する必要はありません (Numpy install メソッドが自動的に実施します)。 パワーユーザーの中には、使用されているBLASがパフォーマンスや、動作、ディスク上のサイズに影響を与えるため、より詳細を知りたいと思っているかもしれません。
+Numpy は他の Python パッケージに依存していませんが、高速な線形代数ライブラリ、一般的には[インテル® MKL](https://software.intel.com/en-us/mkl)、または [OpenBLAS](https://www.openblas.net/) に依存しています。 ユーザーはこれらの線形代数ライブラリのインストールを心配する必要はありません (Numpy install メソッドが自動的に実施します)。 パワーユーザーの中には、使用されているBLASがパフォーマンスや、動作、ディスク上のサイズに影響を与えるため、より詳細を知りたいと思っているかもしれません。
```
Comment (kngwyu) [inline content/ja/install.md:114]:
```suggestion
NumPy は他の Python パッケージに依存していませんが、高速な線形代数ライブラリに依存しています。典型的には、[インテル® MKL](https://software.intel.com/en-us/mkl)や[OpenBLAS](https://www.openblas.net/)がこれにあたります。ユーザーは、これらの線形代数ライブラリのインストールを心配する必要はありません (NumPyのインストール方法に、あらかじめ含まれているためです)。 高度なユーザーは、使用されているBLASがパフォーマンスや、動作、ディスク上のサイズに影響を与えるため、より詳細を知りたがるかもしれません。
```

File: content/ja/install.md [hunk 3]
```diff
@@ -109,55 +109,29 @@ Python パッケージをインストールするための2 つの主要なツ
...
-        インテル® MKL はオープンソースではありません。 通常の使用では問題ではありませんが、 ユーザーが Numpy で構築されたアプリケーションを再配布する必要がある場合、これは 問題が発生する可能性があります。
-      </p>
-    </li>
-    <li>
-      <p spaces-before="0">
-        MKLとOpenBLASの両方とも、 <code>np.dot</code>のような関数呼び出しにマルチスレッドを使用し、スレッド数はビルド時オプションと環境変数の両方で決定されます。 多くの場合、すべての CPU コアが使用されます。 これによりユーザーに予期しないことが起こることがあります。例えばNumPy 自体は、関数呼び出しを自動的に並列化しないことです。 線形代数ライブラリの配列処理は、一般的にはより良いパフォーマンスが得られますが、Daskやscikit-learn、マルチプロセシングなどの別のレベルの並列化を使用している場合などに、逆に悪い結果をもたらすことがあります。
-      </p>
-    </li>
-  </ul>
+- conda-forgeのチャンネルでは、Numpyはダミーの「BLAS」パッケージを使ってビルドされています。 ユーザーがconda-forgeからNumPyをインストールすると、BLASパッケージが実際のライブラリと一緒にインストールされます - デフォルトはOpenBLASですが、MKL(default チャンネルの場合)やBLIS、またはBLASを利用することもできます。
```
Comment (kngwyu) [inline content/ja/install.md:120]:
```suggestion
- conda-forgeのチャンネルでは、NumPyはダミーの「BLAS」パッケージを使ってビルドされています。 ユーザーがconda-forgeからNumPyをインストールすると、BLASパッケージが実際のライブラリと一緒にインストールされます - デフォルトはOpenBLASですが、MKL(default チャンネルの場合)やBLIS、またはBLASを利用することもできます。 
```

File: content/ja/install.md [hunk 4]
```diff
@@ -109,55 +109,29 @@ Python パッケージをインストールするための2 つの主要なツ
...
-      <p spaces-before="0">
-        MKLとOpenBLASの両方とも、 <code>np.dot</code>のような関数呼び出しにマルチスレッドを使用し、スレッド数はビルド時オプションと環境変数の両方で決定されます。 多くの場合、すべての CPU コアが使用されます。 これによりユーザーに予期しないことが起こることがあります。例えばNumPy 自体は、関数呼び出しを自動的に並列化しないことです。 線形代数ライブラリの配列処理は、一般的にはより良いパフォーマンスが得られますが、Daskやscikit-learn、マルチプロセシングなどの別のレベルの並列化を使用している場合などに、逆に悪い結果をもたらすことがあります。
-      </p>
-    </li>
-  </ul>
+- conda-forgeのチャンネルでは、Numpyはダミーの「BLAS」パッケージを使ってビルドされています。 ユーザーがconda-forgeからNumPyをインストールすると、BLASパッケージが実際のライブラリと一緒にインストールされます - デフォルトはOpenBLASですが、MKL(default チャンネルの場合)やBLIS、またはBLASを利用することもできます。 
   
+- OpenBLASのサイズは約30MBですが、MKLパッケージはOpenBLASよりもはるかに大きく、ディスク上の約700MBです。
   
+- MKLは通常、OpenBLASよりも少し速く、よりロバストな結果が得られます。
```
Comment (kngwyu) [inline content/ja/install.md:124]: I guess that 'more robust' means 'numerically more stable'.
```suggestion
- MKLは通常、OpenBLASよりも少し速く、より頑健です。 
```

File: content/ja/install.md [hunk 5]
```diff
@@ -109,55 +109,29 @@ Python パッケージをインストールするための2 つの主要なツ
...
-  </ul>
+- conda-forgeのチャンネルでは、Numpyはダミーの「BLAS」パッケージを使ってビルドされています。 ユーザーがconda-forgeからNumPyをインストールすると、BLASパッケージが実際のライブラリと一緒にインストールされます - デフォルトはOpenBLASですが、MKL(default チャンネルの場合)やBLIS、またはBLASを利用することもできます。 
   
+- OpenBLASのサイズは約30MBですが、MKLパッケージはOpenBLASよりもはるかに大きく、ディスク上の約700MBです。
   
+- MKLは通常、OpenBLASよりも少し速く、よりロバストな結果が得られます。 
+
+インストールサイズ、パフォーマンスとロバスト性に加えて、考慮すべき2つの点があります:
 
+- インテル® MKL はオープンソースではありません。 通常の使用では問題ではありませんが、 ユーザーが Numpy で構築されたアプリケーションを再配布する必要がある場合、これは 問題が発生する可能性があります。
```
Comment (kngwyu) [inline content/ja/install.md:128]:
```suggestion
- インテル® MKL はオープンソースではありません。 通常の使用では問題ではありませんが、 ユーザーが NumPy で構築されたアプリケーションを再配布する必要がある場合、これは 問題が発生する可能性があります。 
```

File: content/ja/install.md [hunk 6]
```diff
@@ -109,55 +109,29 @@ Python パッケージをインストールするための2 つの主要なツ
...
   
+- OpenBLASのサイズは約30MBですが、MKLパッケージはOpenBLASよりもはるかに大きく、ディスク上の約700MBです。
   
+- MKLは通常、OpenBLASよりも少し速く、よりロバストな結果が得られます。 
+
+インストールサイズ、パフォーマンスとロバスト性に加えて、考慮すべき2つの点があります:
 
+- インテル® MKL はオープンソースではありません。 通常の使用では問題ではありませんが、 ユーザーが Numpy で構築されたアプリケーションを再配布する必要がある場合、これは 問題が発生する可能性があります。 
+  
+- MKLとOpenBLASの両方とも、 <code>np.dot</code>のような関数呼び出しにマルチスレッドを使用し、スレッド数はビルド時オプションと環境変数の両方で決定されます。 多くの場合、すべての CPU コアが使用されます。 これによりユーザーに予期しないことが起こることがあります。例えばNumPy 自体は、関数呼び出しを自動的に並列化しないことです。 線形代数ライブラリの配列処理は、一般的にはより良いパフォーマンスが得られますが、Daskやscikit-learn、マルチプロセシングなどの別のレベルの並列化を使用している場合などに、逆に悪い結果をもたらすことがあります。
```
Comment (kngwyu) [inline content/ja/install.md:130]:
```suggestion
- MKLとOpenBLASの両方とも、 <code>np.dot</code>のような関数呼び出しにマルチスレッドを使用し、スレッド数はビルド時オプションと環境変数の両方で決定されます。 多くの場合、すべての CPU コアが使用されます。 これにユーザーにとっては予想外のことかもしれません。NumPy 自体は、関数呼び出しを自動的に並列化しないからです。 自動並列化により、一般にはパフォーマンスが向上しますが、逆にパフォーマンスが悪化する場合もあります。例えば、Daskやscikit-learn、multiprocessingなど別のレベルの並列化を使用している場合です。

**Review by kngwyu — CHANGES_REQUESTED** [review]
Thanks for the fix!
> whole Install page if you have time.

OK, I'm going to create a (possibly) big PR after merging this, to address some typical mistakes at once (e.g., use some regex to replace Numpy by NumPy). Please let me know if another way is better (e.g., crowdin or series of small PRs).

**Comment by rgommers** [comment]
> OK, I'm going to create a (possibly) big PR after merging this, to address some typical mistakes at once (e.g., use some regex to replace Numpy by NumPy). Please let me know if another way is better (e.g., crowdin or series of small PRs).

A PR sounds great, thanks @kngwyu. I think once the dust has settled, Crowdin will be nicer for small updates, but for now let's do whatever makes the most sense.

File: content/ja/install.md (hunk 4 above)
Comment (AtsushiSakai) [reply content/ja/install.md:124]: I feel `より頑健です` is a little bit weird. How is "より安定した解を得られます"?

Comment (kngwyu) [reply content/ja/install.md:124]: Sounds nice 👍🏼

**Comment by AtsushiSakai** [comment]
I think this PR is ready to merge.

**Review by rgommers — APPROVED** [review]
Okay, in it goes - thanks @AtsushiSakai and @kngwyu!

**Pull request merged**